stdlib/Random/test/runtests.jl

```diff
@@ -1063,6 +1063,25 @@ end
     @test maximum(m) <= 0.106
 end
 
+@testset "`shuffle(::NTuple)`" begin
+    @testset "sorted" begin
+        for n ∈ 0:20
+            tup = ntuple(identity, n)
+            @test tup === sort(@inferred shuffle(tup))
+        end
+    end
+    @testset "not identity" begin
+        function is_not_identity_at_least_once()
+            function f(::Any)
+                tup = ntuple(identity, 9)
+                tup !== shuffle(tup)
+            end
+            @test any(f, 1:1000000)
+        end
+        is_not_identity_at_least_once()
```

Comment:
I'm confused... is the goal to clarify what the test ("check not identity at least once") does ? What about
```suggestion
        # shuffle should not be the identity at least once
        tup = ntuple(identity, 9)
        @test any(1:1000000) do _ 
            tup != shuffle(tup)
        end
```